beacon-chain/sync/rpc_beacon_blocks_by_range.go

```diff
@@ -117,6 +112,7 @@ func (s *Service) beaconBlocksByRangeRPCHandler(ctx context.Context, msg interfa
 		// wait for ticker before resuming streaming blocks to remote peer.
 		<-ticker.C
 	}
+	closeStream(stream, log)
```

Comment:
If we have the RPC handler _always_ close the stream instead of resetting, we can remove this. Up to you.